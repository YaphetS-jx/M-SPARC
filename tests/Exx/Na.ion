#=========================
# format of ion file
#=========================
# ATOM_TYPE:   <atom type name> 
# PSEUDO_POT:  <path/to/pseudopotential/>
# N_TYPE_ATOM: <num of atoms of this type>
# ATOMIC_MASS: <mass of atom of this type> #(optional, for MD only)
# COORD:
# <xcoord> <ycoord> <zcoord>
# ...
# RELAX: #(optional)
# <xrelax> <yrelax> <zrelax>
# ...

# this is a comment

ATOM_TYPE:   Na                            # atom type 
N_TYPE_ATOM: 2                             # number of atoms of this type
PSEUDO_POT:  Na_TM.psp   # pseudopotential
COORD:                                     # coordinates follows
    12 15 15
    18 15 15
