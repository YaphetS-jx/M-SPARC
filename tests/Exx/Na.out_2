***************************************************************************
*                      M-SPARC v1.0.0 (Jun 20, 2020)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 10-Oct-2020  Start time: 15:29:56                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 30.000000 30.000000 30.000000 
FD_GRID: 45 45 45
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 15
NSTATES: 2
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/Exx/Na.out_2
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.666667 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/Exx/Na.out_2
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  18
Atom type 1  (valence electrons)   :  Na 9
Pseudopotential                    :  Na.pot
lloc                               :  4
Pseudocharge radii of atom type 1  :  8.00 8.00 8.00
Number of atoms of type 1          :  2
Estimated total memory usage       :  260.02 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1                          NaN        NaN        0.552
