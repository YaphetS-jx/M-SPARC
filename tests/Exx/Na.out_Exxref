***************************************************************************
*                      M-SPARC v1.0.0 (Oct 07, 2020)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 17-Oct-2020  Start time: 16:32:30                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 30.000000 30.000000 30.000000 
FD_GRID: 45 45 45
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 30
NSTATES: 2
EXCHANGE_CORRELATION: LDA
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/Exx/Na.out_30
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.666667 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/Exx/Na.out_30
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  2
Atom type 1  (valence electrons)   :  Na 1
Pseudopotential                    :  Na_TM.psp
lloc                               :  0
Pseudocharge radii of atom type 1  :  8.67 8.67 8.67
Number of atoms of type 1          :  2
Estimated total memory usage       :  260.02 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -9.0152668418E-02        4.874E-01        1.051
2            -1.3440891236E-01        3.277E-01        1.064
3            -1.6645034659E-01        4.074E-01        1.057
4            -1.8361169748E-01        3.459E-01        1.030
5            -1.9274618430E-01        2.344E-01        0.989
6            -1.9721380652E-01        1.594E-01        0.952
7            -1.9917605855E-01        1.214E-01        0.946
8            -1.9997844056E-01        9.919E-02        0.930
9            -2.0036207114E-01        7.139E-02        0.909
10           -2.0052097467E-01        5.696E-02        0.882
11           -2.0055709810E-01        4.963E-02        0.944
12           -2.0043147097E-01        6.685E-02        0.876
13           -2.0053957910E-01        4.120E-02        0.879
14           -2.0056590289E-01        2.796E-02        0.877
15           -2.0062561473E-01        1.796E-02        0.800
16           -2.0064871181E-01        9.513E-03        0.810
17           -2.0065735744E-01        4.794E-03        0.875
18           -2.0065878467E-01        3.269E-03        0.801
19           -2.0065890587E-01        2.611E-03        0.735
20           -2.0065926655E-01        9.013E-04        0.666
Total number of SCF: 20    
1            -2.1491075441E-01        4.034E-02        4.772
2            -2.1501238543E-01        1.821E-02        1.761
3            -2.1503010993E-01        3.108E-03        1.576
4            -2.1503099990E-01        2.017E-03        1.640
5            -2.1503106048E-01        1.045E-03        1.597
6            -2.1503108201E-01        3.771E-04        1.519
7            -2.1503108888E-01        3.986E-04        1.444
8            -2.1503109181E-01        1.996E-04        1.418
9            -2.1503109216E-01        1.167E-04        1.444
10           -2.1503109278E-01        4.654E-05        1.367
11           -2.1503109250E-01        5.683E-05        1.364
12           -2.1503109278E-01        2.958E-05        1.348
13           -2.1503109326E-01        7.824E-06        1.287
14           -2.1503109326E-01        5.676E-06        1.287
15           -2.1503109275E-01        2.948E-06        1.213
16           -2.1503109294E-01        1.604E-06        1.146
17           -2.1503109281E-01        1.361E-06        1.150
18           -2.1503109255E-01        5.110E-07        1.074
Total number of SCF: 18    
1            -2.1536539626E-01        2.618E-03        4.823
2            -2.1536548508E-01        1.456E-03        1.554
3            -2.1536560395E-01        2.171E-04        1.444
4            -2.1536561202E-01        1.736E-04        1.520
5            -2.1536561288E-01        7.441E-05        1.445
6            -2.1536561276E-01        6.051E-05        1.369
7            -2.1536561271E-01        5.109E-05        1.394
8            -2.1536561367E-01        1.586E-05        1.356
9            -2.1536561386E-01        1.746E-05        1.296
10           -2.1536561304E-01        9.338E-06        1.292
11           -2.1536561282E-01        1.515E-06        1.241
12           -2.1536561373E-01        1.849E-06        1.176
13           -2.1536561295E-01        9.377E-07        1.119
Total number of SCF: 13    
1            -2.1539030333E-01        2.160E-04        5.190
2            -2.1539039042E-01        1.182E-04        1.612
3            -2.1539039119E-01        2.422E-05        1.547
4            -2.1539039079E-01        1.303E-05        1.551
5            -2.1539039076E-01        7.736E-06        1.550
6            -2.1539039139E-01        3.571E-06        1.400
7            -2.1539039082E-01        2.002E-06        1.355
8            -2.1539039180E-01        8.421E-07        1.277
Total number of SCF: 8     
1            -2.1539246457E-01        1.895E-05        5.710
2            -2.1539248387E-01        1.097E-05        1.601
3            -2.1539248375E-01        2.594E-06        1.394
4            -2.1539248356E-01        1.548E-06        1.473
5            -2.1539248403E-01        1.133E-06        1.405
6            -2.1539248397E-01        3.846E-07        1.317
Total number of SCF: 6     
1            -2.1539269830E-01        1.146E-06        5.687
2            -2.1539266981E-01        7.452E-07        1.377
3            -2.1539266941E-01        1.836E-07        1.318
Total number of SCF: 3     
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.1539266941E-01 (Ha/atom)
Total free energy                  : -4.3078533882E-01 (Ha)
Band structure energy              : -2.4622695511E-01 (Ha)
Exchange correlation energy        : -2.3731546844E-01 (Ha)
Self and correction energy         : -2.3597317399E+00 (Ha)
Entropy*kb*T                       : -4.0730592109E-10 (Ha)
Fermi level                        : -8.6207968229E-02 (Ha)
Average force                      :  3.0909181506E-03 (Ha/Bohr)
Maximum force                      :  3.0909181506E-03 (Ha/Bohr)
Time for force calculation         :  0.483 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  113.059 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
