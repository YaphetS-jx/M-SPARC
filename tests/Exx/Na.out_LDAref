***************************************************************************
*                      M-SPARC v1.0.0 (Oct 07, 2020)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 10-Oct-2020  Start time: 18:14:18                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 30.000000 30.000000 30.000000 
FD_GRID: 45 45 45
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 30
NSTATES: 2
EXCHANGE_CORRELATION: LDA
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/Exx/Na.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.666667 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/Exx/Na.out
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  2
Atom type 1  (valence electrons)   :  Na 1
Pseudopotential                    :  Na_TM.psp
lloc                               :  0
Pseudocharge radii of atom type 1  :  8.67 8.67 8.67
Number of atoms of type 1          :  2
Estimated total memory usage       :  260.02 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -9.0152668418E-02        4.874E-01        0.846
2            -1.3440891236E-01        3.277E-01        1.029
3            -1.6645034659E-01        4.074E-01        1.039
4            -1.8361169748E-01        3.459E-01        1.047
5            -1.9274618430E-01        2.344E-01        1.001
6            -1.9721380652E-01        1.594E-01        0.970
7            -1.9917605855E-01        1.214E-01        0.958
8            -1.9997844056E-01        9.919E-02        0.960
9            -2.0036207114E-01        7.139E-02        0.922
10           -2.0052097467E-01        5.696E-02        0.884
11           -2.0055709810E-01        4.963E-02        0.960
12           -2.0043147097E-01        6.685E-02        0.887
13           -2.0053957910E-01        4.120E-02        0.892
14           -2.0056590289E-01        2.796E-02        0.888
15           -2.0062561473E-01        1.796E-02        0.815
16           -2.0064871181E-01        9.513E-03        0.826
17           -2.0065735744E-01        4.794E-03        0.890
18           -2.0065878467E-01        3.269E-03        0.819
19           -2.0065890587E-01        2.611E-03        0.745
20           -2.0065926655E-01        9.013E-04        0.674
21           -2.0065926196E-01        9.826E-04        0.673
22           -2.0065932223E-01        3.166E-04        0.614
23           -2.0065933275E-01        3.440E-04        0.669
24           -2.0065933156E-01        3.266E-04        0.679
25           -2.0065932885E-01        3.348E-04        0.600
26           -2.0065932961E-01        2.822E-04        0.545
27           -2.0065933239E-01        2.257E-04        0.533
28           -2.0065933379E-01        1.994E-04        0.555
29           -2.0065933336E-01        2.139E-04        0.529
30           -2.0065933258E-01        2.299E-04        0.534
31           -2.0065933574E-01        1.460E-04        0.457
32           -2.0065933819E-01        4.078E-05        0.542
33           -2.0065933849E-01        1.673E-05        0.610
34           -2.0065933876E-01        1.084E-05        0.534
35           -2.0065933875E-01        6.203E-06        0.530
36           -2.0065933880E-01        4.762E-06        0.463
37           -2.0065933898E-01        3.571E-06        0.399
38           -2.0065933881E-01        1.997E-06        0.314
39           -2.0065933908E-01        1.351E-06        0.313
40           -2.0065933874E-01        1.078E-06        0.314
41           -2.0065933884E-01        4.768E-07        0.315
Total number of SCF: 41    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.0065933884E-01 (Ha/atom)
Total free energy                  : -4.0131867767E-01 (Ha)
Band structure energy              : -2.2627640011E-01 (Ha)
Exchange correlation energy        : -2.3159810699E-01 (Ha)
Self and correction energy         : -2.3597317399E+00 (Ha)
Entropy*kb*T                       : -4.0734595478E-10 (Ha)
Fermi level                        : -8.4635032608E-02 (Ha)
Average force                      :  1.5928234058E-03 (Ha/Bohr)
Maximum force                      :  1.5928234058E-03 (Ha/Bohr)
Time for force calculation         :  0.326 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  30.877 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
