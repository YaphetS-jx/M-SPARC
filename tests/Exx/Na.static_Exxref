***************************************************************************
                            Atom positions                                 
***************************************************************************
Cartesian coordinates of atoms (Bohr):
     12.0000000000      15.0000000000      15.0000000000
     18.0000000000      15.0000000000      15.0000000000
Atomic forces (Ha/Bohr):
      0.0030909182      -0.0000000005      -0.0000000005
     -0.0030909182       0.0000000005       0.0000000005
