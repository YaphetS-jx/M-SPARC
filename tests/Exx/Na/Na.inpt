CELL: 30 30 30
BC: P P P
FD_GRID: 45 45 45
FD_ORDER: 12

EXCHANGE_CORRELATION: GGA_PBE
#EXCHANGE_CORRELATION: LDA_PW
ELEC_TEMP_TYPE: fd
SMEARING: 0.001
TOL_SCF: 1e-6
TOL_POISSON: 1e-8
TOL_PSEUDOCHARGE: 1e-8
NSTATES: 2
PRINT_FORCES: 1
PRINT_ATOMS: 1

CHEB_DEGREE: 30
EXX_FLAG: 1