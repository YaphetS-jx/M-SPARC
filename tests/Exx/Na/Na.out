***************************************************************************
*                      M-SPARC v1.0.0 (Oct 07, 2020)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 02-Nov-2020  Start time: 15:57:39                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 30.000000 30.000000 30.000000 
FD_GRID: 45 45 45
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 30
NSTATES: 2
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/Exx/Na/Na.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.666667 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/Exx/Na/Na.out
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  2
Atom type 1  (valence electrons)   :  Na 1
Pseudopotential                    :  Na_TM.psp
lloc                               :  0
Pseudocharge radii of atom type 1  :  8.67 8.67 8.67
Number of atoms of type 1          :  2
Estimated total memory usage       :  260.02 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -9.2209363784E-02        4.974E-01        0.863
2            -1.3751957771E-01        3.283E-01        1.051
3            -1.6972805963E-01        4.038E-01        1.110
4            -1.8642592429E-01        3.407E-01        1.104
5            -1.9514943987E-01        2.368E-01        1.060
6            -1.9936778101E-01        1.699E-01        1.019
7            -2.0119166079E-01        1.351E-01        1.020
8            -2.0192306654E-01        1.144E-01        1.022
9            -2.0228423761E-01        8.856E-02        1.000
10           -2.0242125382E-01        7.089E-02        0.960
11           -2.0244448850E-01        6.695E-02        1.021
12           -2.0237228014E-01        7.051E-02        0.952
13           -2.0258946109E-01        3.905E-02        0.867
14           -2.0263768921E-01        2.106E-02        0.949
15           -2.0262713871E-01        1.897E-02        0.907
16           -2.0264044733E-01        1.585E-02        0.946
17           -2.0265742427E-01        8.600E-03        0.804
18           -2.0265907100E-01        4.902E-03        0.799
19           -2.0265925646E-01        3.831E-03        0.800
20           -2.0265932856E-01        2.828E-03        0.724
21           -2.0265941271E-01        1.292E-03        0.661
22           -2.0265942121E-01        1.941E-03        0.732
23           -2.0265942115E-01        2.762E-03        0.656
24           -2.0265942465E-01        2.041E-03        0.684
25           -2.0265942524E-01        1.255E-03        0.658
26           -2.0265942580E-01        1.172E-03        0.654
27           -2.0265942526E-01        1.201E-03        0.584
28           -2.0265942534E-01        1.024E-03        0.515
29           -2.0265942527E-01        9.800E-04        0.583
Total number of SCF: 29    
1            -1.9071871297E-01        3.926E-02        6.618
2            -1.9079350167E-01        1.780E-02        2.180
3            -1.9096341677E-01        4.250E-03        1.994
4            -1.9099885783E-01        7.594E-03        2.054
5            -1.9101473612E-01        6.591E-03        2.015
6            -1.9102254219E-01        5.037E-03        1.917
7            -1.9099872020E-01        2.562E-03        1.895
8            -1.9099236439E-01        2.570E-03        1.912
9            -1.9099368871E-01        3.609E-03        1.860
10           -1.9100087683E-01        3.992E-03        1.788
11           -1.9099916749E-01        2.786E-03        1.919
12           -1.9099715341E-01        2.393E-03        1.856
13           -1.9099728743E-01        1.507E-03        1.847
14           -1.9099750517E-01        1.343E-03        1.789
15           -1.9099785656E-01        1.249E-03        1.739
16           -1.9099562701E-01        2.209E-03        1.770
17           -1.9099525277E-01        3.011E-03        1.865
18           -1.9099452452E-01        2.721E-03        1.784
19           -1.9099385882E-01        2.319E-03        1.776
20           -1.9099335829E-01        7.988E-04        1.787
21           -1.9099336480E-01        1.273E-04        1.795
22           -1.9099350208E-01        9.797E-05        1.657
23           -1.9099372910E-01        9.941E-05        1.724
24           -1.9099385314E-01        5.292E-05        1.720
25           -1.9099388138E-01        3.115E-05        1.746
26           -1.9099387578E-01        2.126E-05        1.561
27           -1.9099388060E-01        1.367E-05        1.497
28           -1.9099388536E-01        8.897E-06        1.615
29           -1.9099388544E-01        4.238E-06        1.639
30           -1.9099388368E-01        1.082E-06        1.846
31           -1.9099388403E-01        7.847E-07        1.543
Total number of SCF: 31    
1            -1.9126459736E-01        2.599E-03        7.456
2            -1.9127007278E-01        1.462E-03        2.065
3            -1.9128278619E-01        1.819E-03        1.865
4            -1.9128591847E-01        8.461E-04        1.965
5            -1.9128692401E-01        1.846E-03        2.012
6            -1.9128742094E-01        1.167E-03        1.888
7            -1.9128729154E-01        5.394E-04        1.811
8            -1.9128679369E-01        7.373E-04        1.699
9            -1.9128642753E-01        4.116E-05        1.915
10           -1.9128629283E-01        2.046E-05        1.860
11           -1.9128626099E-01        2.711E-05        1.789
12           -1.9128618341E-01        1.729E-05        1.680
13           -1.9128617444E-01        2.438E-05        1.846
14           -1.9128616936E-01        2.148E-05        1.710
15           -1.9128617489E-01        1.815E-05        1.733
16           -1.9128617654E-01        1.405E-05        1.604
17           -1.9128617805E-01        2.044E-06        1.646
18           -1.9128617872E-01        1.147E-06        1.586
19           -1.9128617581E-01        6.472E-07        1.521
Total number of SCF: 19    
1            -1.9130590593E-01        2.205E-04        7.013
2            -1.9130640551E-01        1.221E-04        2.070
3            -1.9130764826E-01        2.570E-05        1.981
4            -1.9130786346E-01        1.810E-05        2.068
5            -1.9130793252E-01        1.339E-05        1.836
6            -1.9130792107E-01        6.956E-06        1.711
7            -1.9130788430E-01        2.560E-06        1.640
8            -1.9130786382E-01        2.673E-06        1.651
9            -1.9130788021E-01        9.501E-07        1.651
Total number of SCF: 9     
1            -1.9130958022E-01        1.962E-05        7.035
2            -1.9130962621E-01        1.128E-05        1.863
3            -1.9130971400E-01        2.675E-06        1.853
4            -1.9130975196E-01        1.387E-06        1.776
5            -1.9130976284E-01        9.222E-07        1.667
Total number of SCF: 5     
1            -1.9130991785E-01        1.421E-06        6.989
2            -1.9130992154E-01        9.602E-07        1.703
3            -1.9130992840E-01        2.806E-07        1.717
Total number of SCF: 3     
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -1.9130992840E-01 (Ha/atom)
Total free energy                  : -3.8261985681E-01 (Ha)
Band structure energy              : -2.4722669120E-01 (Ha)
Exchange correlation energy        : -1.9109228839E-01 (Ha)
Self and correction energy         : -2.3597317399E+00 (Ha)
Entropy*kb*T                       : -4.0730592769E-10 (Ha)
Fermi level                        : -8.6217033031E-02 (Ha)
Average force                      :  3.4578762606E-03 (Ha/Bohr)
Maximum force                      :  3.4578762606E-03 (Ha/Bohr)
Time for force calculation         :  0.408 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  182.194 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
